src/snippets.ts

```diff
@@ -0,0 +1,83 @@
+import {ElementSnippet} from './editor-service';
+
+const snippets: ElementSnippet[] = [
+  {
+    'tagname': 'html-import',
+    'description': 'Template definition of an HTML import.',
+    'prefix': 'hi',
+    'expandTo': '<link rel="import" href="bower_components/.html">',
+    'expandToSnippet': '<link rel="import" href="${1:bower_components}/${2}/${2}.html">'
+  },
+  {
+    'tagname': 'polymer-element',
+    'description': 'Template definition of a Polymer 2 element',
```

Comment:
I think we should drop this for now, as Polymer 2.0 isn't out yet, and most users initially are likely going to want to write hybrid elements.